tool/site/server/src/routes/auth/callback.ts

```diff
@@ -131,3 +126,8 @@ export default {
 function getQueryString(query: Hapi.RequestQuery) {
   return Object.keys(query).map(key => `${key}=${query[key]}`).join("&")
 }
+
+function separateAuthLogin(request: Hapi.Request) {
```

Comment:
I'd kinda like this to be called `isSeparateAuthLogin` if we get another failure but wouldn't hold up a merge over it